core/arch/arm/kernel/kern.ld.S

```diff
@@ -85,6 +85,10 @@ SECTIONS
 
 	.text : {
 		KEEP(*(.text._start))
+		__identity_map_init_start = .;
+		*(.identity_map .identity_map.* \
+		  .text.__get_core_pos .text.get_core_pos_mpidr)
```

Comment:
> that's weak and we don't want to add that burden on the platforms which overrides `get_core_pos_mpidr()`

Ah, I see. Fair enough. Then I wouldn't mind having a comment here ;-)

As for `__get_core_pos()` I have a slight preference for not making it a special case since there is no real reason, i.e., use `FUNC func , : , .identity_map`. But not a strong opinion.

Reply:
I'll fix